### DISCUSSION TITLE
How to set execution settings when call `planner.create_plan(...)`

### DISCUSSION THREAD

**Discussion by anhnh2002** [microsoft/semantic-kernel#5626] [Q&A] [answered]
How to set execution settings like:
```
"execution_settings": {
      "default": {
        "max_tokens": 512,
        "temperature": 0.0,
        "top_p": 0.0,
        "presence_penalty": 0.0,
        "frequency_penalty": 0.0
      }
```
when creat a plan
```
from planners.sequential_planner.sequential_planner import SequentialPlanner

planner = SequentialPlanner(kernel, service_id)

sequential_plan = await planner.create_plan(goal=ask)
```

**Comment by moonbox3** [comment, marked as answer]
Hi @nguyenhoanganh2002, I see in the following code that the execution settings are hard-coded to read the prompt config from a static json file. The best you can do is configure the number of `max_tokens` via the `SequentialPlannerConfig`.

https://github.com/microsoft/semantic-kernel/blob/f0b1d0df783d0420904a7e07c4957c5ce7d00c48/python/semantic_kernel/planners/sequential_planner/sequential_planner.py#L68C9-L77C14

Are there other config values that you need updated besides the `max_tokens`?
